tests/helpers/test_check_config.py

```diff
@@ -54,13 +54,13 @@ def _assert_warnings_errors(
     error_str = ""
     warning_str = ""
 
-    for idx, error in enumerate(res.errors):
-        assert error == errors[idx]
+    for idx, error in enumerate(errors):
+        assert error == res.errors[idx]
```

Comment:
Please turn it around so that the expected object is on the right and the object to test is on the left.